frontend/src/helper.js

```diff
@@ -273,6 +273,21 @@ export const getLoginErrorFromResponse = (status, data) => {
   }
 };
 
+export const validatePassword = (newPassword) => {
```

Comment:
Please look at the regex for password criteria
const passwordRegex = /^(?=.*[A-Za-zÀ-ÿ].*)(?=.*[0-9].*)([A-Za-zÀ-ÿ0-9-_!?@#$&*,.:/()[\] ])*$/;